### ISSUE TITLE
Reconsider e.printStackTrace and add logging

### ISSUE THREAD

**Issue by grobmeier** [wildbit/postmark-java#9]
At this line:
https://github.com/wildbit/postmark-java/blob/master/src/main/java/com/wildbit/java/postmark/Postmark.java#L51

the stack trace is written to console. In clustered environments this message could get lost, as the console output is disabled. Logging messages are usually directed using a logging framework.

One dependency is already using slf4j:

```
[INFO] +- eu.medsea.mimeutil:mime-util:jar:2.1.3:compile
[INFO] |  +- org.slf4j:slf4j-api:jar:1.5.6:compile
[INFO] |  +- org.slf4j:slf4j-log4j12:jar:1.5.6:runtime
[INFO] |  \- log4j:log4j:jar:1.2.14:runtime
```

Unfortunately it's an older version. Another option would be to use JUL directly. However, usually slf4j is being used as it's compatible with all major frameworks and is also offering basic logging for those who don't want to include them.

**Comment by ibalosh** [comment]
agreed, will consider it in next release

tx!

**Comment by ibalosh** [comment]
should be resolved in 1.1.4 version
